### PR TITLE
Do not redirect if CLI mode

### PR DESCRIPTION
**Pull request by vitalyzhakov**
@joshp23 , thank you for solving https://github.com/joshp23/YOURLS-Expiry/issues/15!
But it does not work for me properly. It removes one link and die with html output.

I have added some checks. Can your merge it?

### CODE REVIEW

**Review by joshp23 — CHANGES_REQUESTED** [review]
Please change `EXPIRY_CONSOLE_MODE` to `EXPIRY_CLI` and update version in `plugin.php` to `2.1.1`

**Comment by vitalyzhakov** [comment]
@joshp23 , OK, files changed.
Additionally, I deleted `// ini_set('max_execution_time', 3000);`, because by default php from CLI sets it to 0 https://www.php.net/manual/en/info.configuration.php#ini.max-execution-time

File: expiry/bin/prune.inc.php [hunk 1]
```diff
@@ -1,13 +1,12 @@
 <?php
 // XXX Uncomment these lines if you run into memory limit failures
-// ini_set('max_execution_time', 3000);
```
Comment (joshp23) [inline expiry/bin/prune.inc.php]: revert this change, and all of the subsequent commits, and I will merge. The other issues (adjusting sql queries, adding the link limit) would please be a separate request. There is another way I may go with addressing the sql issue.

**Comment by joshp23** [comment]
I put the `// ini_set('max_execution_time', 3000);` back. It seems useful to me to keep it there for the reverse of your issue, if folks require the limit for whatever reason. Also, please remove the 3 commits addressing memory consumption from this pull request, I would put them in another branch and open another pull request for them, specifically. I may go in another direction to address that issue, and would not like to commit that at this time.

**Comment by vitalyzhakov** [comment]
@joshp23 , ok, no problem)

**Pull request merged**